aboutme.html

```diff
@@ -0,0 +1,48 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
```

Comment:
you dont need to use `./` in the href. just doing `css/normalise.css` will work fine